.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,18 @@
+<!--
+- Please fill in this template according to the PR you're about to submit
+- Features and deprecations must be submitted against the master branch
+- Bug fixes must be submitted against the lowest branch where they apply
+  (lowest branches are regularly merged to upper ones so they get the fixes too)
+-->
+
 | Q             | A
 | ------------- | ---
-| Branch?       | "master" for new features / 2.7, 2.8 or 3.1 for fixes
```

Comment:
bad idea IMHO: makes more things to remove, both for newcomers and oldcomers. And failling to remove the surrounding `<!-- -->` will make the field blank. And you know this will happen :)